rtc/configuration.md

```diff
@@ -9,8 +9,6 @@ keywords: rtc configuration
 {% assign beta_feature = "The RTC plugin" %}
 {% assign pre-release_type = "Closed Beta" %}
 {% include misc/beta-note.md %}
-{{site.requires_5_6v}}<br>
```

Comment:
Why was this removed? This is needed, otherwise people may try to use it with an older version (hybrid mode is a thing on cloud).

I can sort of understand the `premiumplugin` part since this applies to closed beta.

Reply:
my understanding is that the plugin is only served on the nightly build for Tiny Cloud and the minimum version may change before the feature is released to the public.

Reply:
Yeah, the minimum version will likely change, however at the moment I could use cloud to load RTC and then still use TinyMCE 5.3 (or lower) locally since we can use hybrid mode. So I really think we need this, even if we have to update it later on to say 5.7 or whatever.

Reply:
The beta access doc already tell you to use the 5-dev channel.